src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -866,6 +868,7 @@ void readEvents(final List<O> evtOps) {
 				var evtReader_ = evtReaderPool.poll();
 				if(null == evtReader_) {
 					evtReader_ = clientFactory.createReader("reader-" + (System.nanoTime()), evtReaderGroupName, evtDeserializer, evtReaderConfig);
+					Loggers.MSG.info("{}: created a new reader in {} {}", stepId, Thread.currentThread().getName(), evtReader_.toString());
```

Comment:
What's the purpose of this message in the info level? How could it be used for the log analysis? 
I just want to make sure we're not filling the info level with the stuff that'd be useful mostly during the debug process.

Reply:
As stated in the doc, it's user's responsibility to match the amount of readers to amount of segments in the stream while working with FixedSegmentCount. We are using reader per java thread approach and at the same time the most efficient approach is when amount of readers equals to amount of segments. So it's done to help users understand if they have correctly configured the tool.